plugins/ja/qml/keys/CursorKey.qml

```diff
@@ -51,6 +51,9 @@ ActionKey {
     }
 
     onPressAndHold: {
-        return;
+        fullScreenItem.cursorSwipe = true;
+        if(rightSide) {
```

Comment:
Can you please fix the whitespace here too (should be `if (rightside) {`), and clean up the commits with `git rebase -i HEAD~4` (since I presume this will mean you'd have 4 commits after fixing the whitespace), and squash the latter 3 into the first, with a single commit message relevant to the issue being fixed (please include the `Fixes #98` in the commit message extended description)? Thanks.

Reply:
oof missed that one will do.